src/generic/complex_matrices.cc

```diff
@@ -899,8 +899,8 @@ namespace oomph
       }
 
       // build
-      unsigned long this_nnz = this->nnz();
-      result.build_without_copy(value, column_index, row_start, this_nnz, m, n);
+      unsigned long nnz = this->nnz();
```

Comment:
Change to `n_nz`.

Reply:
Agree with Puneet and this specific case is also discussed in the coding conventions: No underscore in function names that access the "number of things" in a container, and then use the underscore to store the result from calling that function.

Reply:
Sorted. I been referring to the coding conventions from the old website http://oomph-lib.maths.man.ac.uk/doc/coding_conventions/html/index.html. Have they changed since?

Reply:
Not been updated, but it's true that the intended usage was implied rather than spelled out. I'm about to commit the following
modification:

 \b Notes: (i) No underscore between the "n" and the container's name;
 You can then use the underscore for the name of the local variable
 that stores the value, e.g.
 \c
 unsigned \c n_element \c = \c my_mesh_pt->nelement();